app/_hub/peter-evans/paseto/index.md

```diff
@@ -111,7 +111,7 @@ params:
 
 ---
 
-## Installation
+### Installation
```

Comment:
This should stay at H2

Reply:
Ok.
Should the `Usage` header stay at H2 as well?